scripts/data/supersearch_fields.json

```diff
@@ -2865,29 +2811,27 @@
                     "dynamic": "true",
                     "properties": {
                         "classification_version": {
-                            "type": "string",
-                            "analyzer": "keyword"
+                            "type": "keyword"
                         },
                         "classification_data": {
-                            "type": "string"
+                            "type": "text"
```

Comment:
All I remember is that it has text about the classification. But that is not exposed anywhere, so it doesn't really matter.